Tests/Runtime/Annotations/PositionAnnotationTest.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2023 Koji Hasegawa.
+// This software is released under the MIT License.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Cysharp.Threading.Tasks;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+
+namespace TestHelper.Monkey.Annotations
+{
+    [UnityPlatform(RuntimePlatform.OSXEditor, RuntimePlatform.WindowsEditor, RuntimePlatform.LinuxEditor)]
+    [TestFixture]
+    public class PositionAnnotationTest
+    {
+        [SetUp]
+        public async Task SetUp()
+        {
+#if UNITY_EDITOR
+            await UnityEditor.SceneManagement.EditorSceneManager.LoadSceneAsyncInPlayMode(
+                "Packages/com.nowsprinting.test-helper.monkey/Tests/Scenes/Annotations.unity",
+                new LoadSceneParameters(LoadSceneMode.Single));
```

Comment:
Please bump test-helper package to v0.3.0 in your project, and use LoadSceneAttribute.

See example:
https://github.com/nowsprinting/test-helper.monkey/commit/84566c1d06ff7b297289e9268cda78a66db8b1f7

Reply:
Fixed at [6a14839](https://github.com/nowsprinting/test-helper.monkey/pull/47/commits/6a14839ccc6739517ea095f9616887700a248883).

Reply:
Thanks!

Please actions follow:

- remove load scene in SetUp method
- remove limitation running in Editor by UnityPlayformAttribute

Reply:
Ahh, I'm sorry. Fixed at 8aed084.

Reply:
Thanks!